CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# How to contribute
+
+Contributions to ScalaBridge are welcomed and encouraged - **without input from volunteers we would have nothing!** 
+
+By participating in this project you agree to abide by the [Bridge Foundry](http://bridgefoundry.org/) [code of conduct](http://bridgefoundry.org/code-of-conduct/).
+
+You can contribute to ScalaBridge by either:
+
+1. **Creating an `issue`** to suggest an enhancement, identify a 'bug', or make another request, or
+2. **Creating a `pull request`** to resolve an issue by revising existing content or creating new content.  
+
+* To create `issues` or make `pull requests` you need a [GitHub account](#setup).
```

Comment:
Let's remove the bullet here to make this a plain paragraph. And you'll need to update the link since that section moved to another file.